### PR TITLE
Update UG for Jiale's tasks

### PR DESCRIPTION
**Pull request by Singa-pirate**
Update UG for Jiale's tasks.

- Remove traces of AB3 UG
- Add description and formatting

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -145,28 +70,95 @@ as a link to your contact at index 2.`
 
 ![add link success](images/add-remark/add-link.png)
 
-### Listing all persons : `list`
 
-Shows a list of all persons in the address book.
 
-Format: `list`
+### **Assign priority levels:** **`add /priority /index`**
```
Comment (nknguyenhc) [inline docs/UserGuide.md]: This line should not have bold formatting.
The text would alr have been bolded by the hashes.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -145,28 +70,95 @@ as a link to your contact at index 2.`
...
+
+`add /priority [priority level] /index [index]`
+
+- `add /priority h /index 100000`
+
+`OOPS, no matching contact with index 100000.`
+
+
+
+### **Add tag to a contact: **  **`add /tag /index`**
```
Comment (nknguyenhc) [inline docs/UserGuide.md]: Same here, text should not be bold formatted.

**Review by nknguyenhc — CHANGES_REQUESTED** [review]
Looks good in general, please correct the formatting inconsistencies.

**Comment by Singa-pirate** [comment]
> Looks good in general, please correct the formatting inconsistencies.

Fixed the formatting inconsistencies, please review again, thank you.

**Review by nknguyenhc — APPROVED** [review]
LGTM

**Pull request merged**